src/api/services/NotificationAPI.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * Italia PagoPA Proxy
+ * Cittadinanza Digitale PagoPA services
+ */
+
+import * as express from "express";
+import fetch from "node-fetch";
+import { CONFIG } from "../../Configuration";
+import { NotificationSubscriptionRequestType } from "../../enums/NotificationSubscriptionType";
+import { INotificationSubscriptionResponse } from "../types/NotificationSubscriptionResponse";
+
+// Notification Service for PagoPA communications
+export class NotificationAPI {
+  // Retrieve wallet containing payment methods
+  public static updateSubscription(
+    res: express.Response,
+    errorCallback: (res: express.Response, errorMsg: string) => void,
+    successCallback: (
+      res: express.Response,
+      notificationSubscriptionResponse: INotificationSubscriptionResponse
+    ) => void,
+    fiscalCode: string,
+    requestType: NotificationSubscriptionRequestType
+  ): void {
+    const url =
+      CONFIG.PAGOPA.HOST +
+      ":" +
+      String(CONFIG.PAGOPA.PORT) +
+      CONFIG.PAGOPA.SERVICES.NOTIFICATION_UPDATE_SUBSCRIPTION;
+    const body = {
+      timestamp: Date.now(),
+      requestId: Math.floor(Math.random() * 99999999999999999999),
```

Comment:
`Number.MAX_SAFE_INTEGER` ?

Reply:
Ok. Changed into next push

Reply:
to save time, you can just 👍 the comment instead of writing that sentence on every comment :)

![image](https://user-images.githubusercontent.com/48206/40125859-37f016b6-592c-11e8-868c-c9a844eac357.png)